### PR TITLE
Merge into locomotion the integration with stripe

### PR DESCRIPTION
**Pull request by OfryL**
## ℹ️ Change description:
https://autofleet.atlassian.net/browse/AUT-7693

### CODE REVIEW

File: locomotion-sdk/src/LocomotionRouter.js [hunk 1]
```diff
@@ -1,25 +1,38 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 import params from 'react-native-config';
+import { initStripe } from '@stripe/stripe-react-native';
 
 import { MainProvider } from './context/main';
 // import StorybookUI from './storybook';
+//import Router, { MainRouter } from './pages';
```
Comment (oriefrati) [inline locomotion-sdk/src/LocomotionRouter.js]: remove

File: locomotion-sdk/src/pages/ActiveRide/RideDrawer/index.js [hunk 2]
```diff
@@ -115,13 +124,26 @@ const RideDrawer = ({
...
     }
   }, [futureRides])
+
+  useEffect(() => {
+    if(usePayments.paymentMethods) {
+      if(usePayments.paymentMethods.length > 0) {
+        setAllowRideOrder(true);
+      } else {
+        setAllowRideOrder(false);
+      }
```
Comment (oriefrati) [inline locomotion-sdk/src/pages/ActiveRide/RideDrawer/index.js]: lets combine to one if

File: server/src/api/v1/admin/service-hours/index.ts [hunk 3]
```diff
@@ -26,7 +26,7 @@ router.post('/', async (req, res) => {
 
 router.delete('/:slotId', async (req, res) => {
   const { slotId } = req.params;
-  let slots = [];
+  let slots: number = null;
```
Comment (oriefrati) [inline server/src/api/v1/admin/service-hours/index.ts:29]: should be array (as the previous default value)

File: server/src/api/v1/admin/settings/index.ts [hunk 4]
```diff
@@ -62,7 +62,7 @@ router.post('/', async (req, res) => {
 
 router.get('/:settingId', async (req, res) => {
   const { settingId } = req.params;
-  let foundSettings = [];
+  let foundSettings: Setting = null;
```
Comment (oriefrati) [inline server/src/api/v1/admin/settings/index.ts]: set default to array, null would break the code

**Review by oriefrati — APPROVED** [review]

File: server/src/api/v1/admin/service-hours/index.ts (hunk 3 above)
Comment (OfryL) [reply server/src/api/v1/admin/service-hours/index.ts:29]: is ok, slots is number of deleted slots.

**Pull request merged**